healpy/sphtfunc.py

```diff
@@ -1015,6 +1027,12 @@ def smoothing(
             output_map.append(alm2map(alm, nside, pixwin=False, verbose=verbose))
         output_map = np.array(output_map)
     output_map[masks] = UNSEEN
+    
+    if nest:
+        ordering = "NESTED"
+        output_map = pixelfunc.reorder(output_map, inp=None, out=None, r2n=True, n2r=False)
+    else:
+        ordering = "RING" 
```

Comment:
we probably dont need to redefine `ordering` here.

Reply:
Yes true, thank you! I have removed the "ordering" and also the else condition which is not essential I suppose.